modules/other-integrations/pages/execute-dbt-teradata-transformations-in-airflow-with-cosmos.adoc

```diff
@@ -0,0 +1,248 @@
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+Linux::
++
+--
+[source, bash]
+----
+AIRFLOW_VERSION=2.9.2
+PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
+CONSTRAINT_URL="https://raw.githubusercontent.com/apache/airflow/constraints-${AIRFLOW_VERSION}/constraints-${PYTHON_VERSION}.txt"
+pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+----
+--
+====
+
+3. Install the Apache Airflow Teradata provider stable version from the corresponding PyPI repository.
++
+[source, bash]
+----
+pip install "apache-airflow-providers-teradata"
+----
+
+== Install Astronomer Cosmos
+[source, bash]
+----
+pip install "astronomer-cosmos"
+----
+
+== Install dbt
+1. Create a new python environment to manage dbt and its dependencies. Activate the environment:
++
+[tabs]
+====
+Windows::
++
+--
+[source, bash]
+----
+python -m venv env
+source env/bin/activate
+----
+--
+MacOS::
++
+--
+[source, bash]
+----
+python3 -m venv env
+source env/bin/activate
+----
+--
+Linux::
++
+--
+[source, bash]
+----
+python3 -m venv env
+source env/bin/activate
+----
+--
+====
+
+2. Install `dbt-teradata` and `dbt-core` modules:
++
+[source, bash]
+----
+pip install dbt-teradata dbt-core
+----
+
+== Setup dbt project
+
+1. Clone the jaffle_shop repository and cd into the project directory:
++
+[source, bash]
+----
+git clone https://github.com/Teradata/jaffle_shop-dev.git jaffle_shop
+cd jaffle_shop
+----
+2. Make a new folder, dbt, inside $AIRFLOW_HOME/dags folder. Then, copy/paste jaffle_shop dbt project into $AIRFLOW_HOME/dags/dbt directory
++
+[source, bash]
+----
+mkdir -p $AIRFLOW_HOME/dags/dbt/
+cp -r jaffle_shop $AIRFLOW_HOME/dags/dbt/
+----
+
+== Configure Apache Airflow
+
+1. Configure the listed environment variables to activate the test connection button, preventing the loading of sample DAGs and default connections in Airflow UI.
++
+[source, bash]
+  export AIRFLOW__CORE__TEST_CONNECTION=Enabled
+  export AIRFLOW__CORE__LOAD_EXAMPLES=false
+  export AIRFLOW__CORE_LOAD_DEFAULT_CONNECTIONS=false
+
+2. Define the path of jaffle_shop project as an environment variable `dbt_project_home_dir`.
++
+[source, bash]
+export dbt_project_home_dir=../../jaffle_shop
++
+NOTE: You might need to change `/../../` to the specific path where you cloned the jaffle_shop project.
+
+3. Define the path to the virtual environment where dbt-teradata was installed as an environment variable `dbt_venv_dir`.
+[source, bash]
+set dbt_venv_dir=/../../dbt_env/bin/dbt
++
+NOTE: You might need to change `/../../` to the specific path where the virtual environment is located.
+
+== Start Apache Airflow web server
+1. Run airflow web server
++
+[source, bash]
+----
+airflow standalone
+----
+2. Access the airflow UI. Visit https://localhost:8080 in the browser and log in with the admin account details shown in the terminal.
+
+== Define Apache Airflow connection to Vantage Cloud Lake
```

Comment:
This one let's make it more generic, assume that you are working with CSAE, this is applicable to lake as it appears in the compute clusters guide, but here we don't have all the section regarding  the SQL needed for creating the user, if we just point that you need a teradata environment and you can get one on CSAE we save all of that part. We just need to modify this section then to use the CSAE host, user and password.

Reply:
Modified as more generic.